.github/workflows/pr-welcome-msg.yml

```
@@ -24,6 +24,8 @@ jobs:
             - **review please** to notify leapp developers of review request
             - **/packit copr-build** to submit a public copr build using packit
 
+            Packit will automatically schedule regression tests for this PR's build and leapp\*master\*. If you need a different version of leapp from PR#42, use `/packit test oamg/leapp#42`
```

```suggestion
            Packit will automatically schedule regression tests for this PR's build and latest upstream (devel) build of leapp. If you need a different version of leapp from PR#42, use `/packit test oamg/leapp#42`
```

@fernflower   I prefer to rather skip the name of the master/main/upstream branch as we are anyway enforced to change it soon to something else.

---

Ok, I will use `latest upstream build` instead, thanks

---

Done everywhere